go.mod

```diff
@@ -0,0 +1,40 @@
+module github.com/improbable-eng/grpc-web
+
+require (
+	github.com/beorn7/perks v0.0.0-20180321164747-3a771d992973 // indirect
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/fsnotify/fsnotify v1.4.7 // indirect
+	github.com/gogo/protobuf v1.1.1 // indirect
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b // indirect
+	github.com/golang/protobuf v1.2.0
+	github.com/gorilla/websocket v1.2.0
+	github.com/grpc-ecosystem/go-grpc-middleware v1.0.0
+	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
+	github.com/hpcloud/tail v1.0.0 // indirect
+	github.com/matttproud/golang_protobuf_extensions v1.0.1 // indirect
+	github.com/mwitkow/go-conntrack v0.0.0-20161129095857-cc309e4a2223
+	github.com/mwitkow/grpc-proxy v0.0.0-20171120185045-67591eb23c48
+	github.com/onsi/ginkgo v1.6.0 // indirect
+	github.com/onsi/gomega v1.4.1 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/prometheus/client_golang v0.8.0
+	github.com/prometheus/client_model v0.0.0-20180712105110-5c3871d89910 // indirect
+	github.com/prometheus/common v0.0.0-20180801064454-c7de2306084e // indirect
+	github.com/prometheus/procfs v0.0.0-20180725123919-05ee40e3a273 // indirect
+	github.com/rs/cors v1.5.0
+	github.com/sirupsen/logrus v1.0.4
+	github.com/spf13/pflag v1.0.0
+	github.com/stretchr/testify v1.1.4
+	golang.org/x/crypto v0.0.0-20180910181607-0e37d006457b // indirect
+	golang.org/x/net v0.0.0-20171207222016-dc871a5d77e2
+	golang.org/x/sync v0.0.0-20180314180146-1d60e4601c6f // indirect
+	golang.org/x/sys v0.0.0-20180909124046-d0be0721c37e // indirect
+	golang.org/x/text v0.3.0 // indirect
+	google.golang.org/genproto v0.0.0-20180831171423-11092d34479b // indirect
+	google.golang.org/grpc v1.14.0
```

Comment:
I don't think this will work with `grpcwebproxy`. It was locked to `v1.8.0` in `Gopkg.toml`.

Reply:
Strange, it did get the initial versions from Gopkg.toml. I only bumped the above mentioned (to get a cleaner go.mod file).

Reply:
Should I also update the `grpcwebproxy` to work with 1.14.0 if that is the case?